src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource.cs

```diff
@@ -193,21 +184,6 @@ private object State
                 }
             }
 
-            private bool UpToDate
-            {
-                get
-                {
-                    _workQueue.AssertIsForeground();
-                    return _upToDate_doNotAccessDirectly;
-                }
-
-                set
-                {
-                    _workQueue.AssertIsForeground();
-                    _upToDate_doNotAccessDirectly = value;
-                }
-            }
```

Comment:
the previous logic was more complex and required us to keep track if there were outstanding operations in flight that we had to cancel and refire.  We also didn't want to recompute if our comptuation was done and no more events had been heard.  But now, we're not implementing IAccurateTagger anymore, so we don't need to provide those semantics.  It's fine to return stale results all the time *except* for the two cases in outlinign we need it.